modules/foundational/single.yaml

```diff
@@ -0,0 +1,80 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Sysdig Secure Foundational Onboarding
+
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: "Sysdig Settings (Do not change)"
+        Parameters:
+          - RoleName
+          - ExternalID
+          - TrustedIdentity
+
+    ParameterLabels:
+      RoleName:
+        default: "Role Name (Sysdig use only)"
+      ExternalID:
+        default: "External ID (Sysdig use only)"
+      TrustedIdentity:
+        default: "Trusted Identity (Sysdig use only)"
+
+Parameters:
+  RoleName:
+    Type: String
+    Description: The read-only IAM Role that Sysdig will create
+  ExternalID:
+    Type: String
+    Description: Sysdig generated token that proves you own this account
+  TrustedIdentity:
+    Type: String
+    Description: The Role in Sysdig's AWS Account with permissions to your account
+
+Resources:
+  CloudAgentlessRole:
```

Comment:
We would want two roles right, one for CSPM and one for Onboarding?

Reply:
yep. This will be a long running WIP.